fulp_modules/bloodsuckers/code/antagonists/bloodsucker_datum.dm

```diff
@@ -266,7 +266,11 @@
 			H.dna.remove_mutation(CLOWNMUT)
 			to_chat(H, "As a vampiric clown, you are no longer a danger to yourself. Your clownish nature has been subdued by your thirst for blood.")
 	// Physiology
-	CheckVampOrgans() // Heart, Eyes
+	CheckVampOrgans() // Heart, Tongue
+	// Makes eyes flash-immune
```

Comment:
why are you saying that this makes their eyes flash-immune when it actually makes them flash-weak?

Reply:
because my brain was wandering when I wrote it, thanks for the catch